doc/nrf/ug_radio_fem.rst

```diff
@@ -127,12 +133,57 @@ This feature uses a model to compensate the FEM gain for the following external
 The model assumes that the FEM supply voltage is constant.
 To provide the value of this voltage to the MPSL subsystem, use the :kconfig:option:`CONFIG_MPSL_FEM_POWER_VOLTAGE` option.
 
+This model is enabled by default, provided that nRF21540 GPIO/SPI is enabled and selected. Refer to :ref:`ug_radio_fem_nrf21540_spi_gpio` for instructions how to do that.
```

Comment:
```suggestion
This model is enabled by default, provided that nRF21540 GPIO+SPI is enabled and selected.
To enable and select nRF21540 GPIO+SPI, see :ref:`ug_radio_fem_nrf21540_gpio_spi`.
```

Reply:
You've suggested to rename `ug_radio_fem_nrf21540_spi_gpio` to `ug_radio_fem_nrf21540_gpio_spi`, but this section is already referenced in other parts of the documentation and I'd prefer not to change it. Is that acceptable?